### ISSUE TITLE
External connection verification

### ISSUE THREAD

**Issue by KarenChristensen** [10up/distributor#1028]
### Describe your question

For no obvious reason (I've checked that I'm logging in with an Admin user name/password), I am getting the following message. Have tried this over and over again, no luck. Connection within multisite is fine. And the pull the other direction (from Berkshirepublishing.com into Karenchristensen.org) is working. But not the other way around.
Limited connection established.
Authentication failed due to invalid credentials.
Push distribution unavailable.
Pull distribution limited to basic content, i.e. title and content body.
Post types permissions

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by KarenChristensen** [comment]
I have found that if I go to edit a post, the Distributor link at the top then gives me a popup option to distribute it, and this is working, it seems.

**Comment by peterwilsoncc** [comment]
Welcome @KarenChristensen and thank you for logging this.

Just to confirm, is this what you are seeing on the external connection screen?

<img width="1215" alt="Screen Shot 2023-03-15 at 3 03 00 pm" src="https://user-images.githubusercontent.com/519727/225203550-fffeda34-27b6-40a5-8689-ef37691e83f8.png">

_For the screen shot, I've modified the account so it is expected (the user 'subscriber' does not have publishing permissions). I understand your report is for a user that is permitted to publish._

> the Distributor link at the top then gives me a popup option to distribute it, and this is working, it seems.

I presume this is in contradiction of the message you see on the external connection screen? 

Thanks again for the report,
Peter

**Comment by KarenChristensen** [comment]
Yes, that’s the message I was seeing. Attached is how I worked around it, though it was much slower: the connection seems to be there. I love this plugin – for a small publishing company headed by a writer, it is just what we need. Thanks, Karen.

Karen Christensen
Berkshire Publishing Group LLC
Email: ***@***.******@***.***>
--
Subscribe to Karen’s Letter: Thinking globally https://karenchristensen.substack.com
